cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -96,14 +97,25 @@ public void tearDown() throws Exception {
     // TODO: check metrics, streams, etc.
   }
 
-  protected TestManager getTestManager() {
+  protected ClientConfig getModifiableClientConfig(ClientConfig initialClientConfig, Id.Namespace namespace) {
```

Comment:
So this method exists to replace a one liner?

`new ClientConfig.Builder(otherConfig).setNamespace(namespace).build()`

Seems a bit unnecessary to me.